awx/main/models/workflow.py

```diff
@@ -438,9 +437,22 @@ def notification_templates(self):
                                               .filter(unifiedjobtemplate_notification_templates_for_started__in=[self]))
         success_notification_templates = list(base_notification_templates
                                               .filter(unifiedjobtemplate_notification_templates_for_success__in=[self]))
+        approval_notification_templates = list(base_notification_templates
+                                               .filter(unifiedjobtemplate_notification_templates_for_approvals__in=[self]))
+        # Get Organization NotificationTemplates
+        if self.organization is not None:
+            error_notification_templates = set(error_notification_templates + list(base_notification_templates.filter(
```

Comment:
Prior to this commit, it looks like the `failure`, `success`, and `started` notifications *don't* inherit from the Organization.  I think for the purposes of _this_ change, we probably don't want to introduce that behavior change.

We *do*, however, want to keep the line you added for inheriting *approval* notifications from the org:

```python
approval_notification_templates = set(
    approval_notification_templates + list(base_notification_templates.filter(
        # etc...
```

Reply:
> Prior to this commit, it looks like the failure, success, and started notification templates for workflows don't inherit from the Organization. Maybe that could be considered a bug

fwiw, sounds like a bug to me

Reply:
@matburt @wenottingham can you confirm that this is a bug, or is there a *reason* for this behavior?

Reply:
Is this for workflows with an explicit org?

Reply:
Right, we think we've discovered that WFJTs w/ an explicit org _will not_ inherit Notification Templates from that org (at all).

Reply:
Let's treat is as a bug and track it as separate work @beeankha @jladdjr